static/styles/recent_topics.css

```diff
@@ -287,11 +322,21 @@
            as new messages arrive from the server. */
         .recent_topic_stream {
             width: 25%;
-            padding: 8px;
+            padding: 8px 0 8px 8px;
         }
 
         .recent_topic_name {
             width: 40%;
+
+            .line_clamp {
+                /* As display: -webkit-box is not valid for */
+                /* display property in stylelint library */
+                /* stylelint-disable-next-line value-no-vendor-prefix */
```

Comment:
This explanation is not particularly satisfying. I believe StyleLint bans using prefixes like `-webkit-box` because they are not going to work on Firefox; see: https://caniuse.com/?search=webkit-box

You can test on Firefox to see if this is indeed broken there, but generally our preference will be to use a different CSS mechanism that is part of the standard.

Reply:
I have tested with firefox and is working as expected.

Reply:
@timabbott I have tested both on firefox and chrome, line clamping using webkit is working fine. Can we use it in our final versions with suitable comments?

Reply:
I'll change the comment and integrate this, but please start a #frontend thread about whether there's a better way to do line clamping without this `webkit-box` feature that is webkit-specific.

Reply:
Yes, I will start the conversation there.